frontend/Makefile

```diff
@@ -1,5 +1,5 @@
 APP_PORT ?= 3000
-ENV_FILE ?= .env
+ENV_FILE ?= .env.local
```

Comment:
It should also work with the .env file, but NextJS has built-in support for loading environment variables. 
Check the documentation: [NextJS Documentation](https://nextjs.org/docs/pages/building-your-application/configuring/environment-variables#loading-environment-variables)
I have tested using `.env` instead of `.env.local` and it works as expected. I change it to `.env` to fix the Makefile 👌